module/documents/item.mjs

```diff
@@ -2323,10 +2328,20 @@ export default class Item5e extends Item {
       name: `${game.i18n.localize("DND5E.SpellScroll")}: ${itemData.name}`,
       img: itemData.img,
       system: {
-        "description.value": desc.trim(), source, actionType, activation, duration, target, range, damage, formula,
+        description: {value: desc.trim()}, source, actionType, activation, duration, target, range, damage, formula,
         save, level, attackBonus
       }
     });
+
+    /**
+     * A hook event that fires after the item data for a scroll is created but before the item is returned.
+     * @function dnd5e.createScrollFromSpell
+     * @memberof hookEvents
+     * @param {Item5e|object} spell       The spell or item data triggering this function.
+     * @param {object} itemData           The item data of the spell triggering this function.
```

Comment:
I'm not sure there is much reason to pass both the spell as an document and an object. We aren't doing any actual modifications to `itemData` so consumers of this hook could just get it themselves from the first property.